resources/assets.py

```diff
@@ -2348,6 +2391,19 @@ def slab_loot(rm: ResourceManager, loot: str) -> BlockContext:
         }]
     })
 
+def mold_model(rm: ResourceManager, mold_item_location: str, pattern: str):
```

Comment:
I may be remembering wrong but I think we have a `rm.custom_block_model` call to avoid this `rm.write` invocation